pkg/preparer/setup.go

```diff
@@ -376,11 +384,15 @@ func New(preparerConfig *PreparerConfig, logger logging.Logger) (*Preparer, erro
 		logExec = runit.DefaultLogExec()
 	}
 
-	var finishExec []string
-	if len(preparerConfig.FinishExec) > 0 {
-		finishExec = preparerConfig.FinishExec
-	} else {
-		finishExec = pods.DefaultFinishExec
+	finishExec := pods.NopFinishExec
+	podProcessReporterLogger := logger.SubLogger(logrus.Fields{
+		"component": "PodProcessReporter",
+	})
+
+	// Returns nil if config is unset
```

Comment:
I feel not at all strongly about this, but: did you consider using a second return value that signals whether or not this should be enabled? `nil` as a sentinel value is fine in most cases but I thought it worth mentioning.